src/liballoc/collections/btree/set.rs

```diff
@@ -557,7 +557,7 @@ impl<T: Ord> BTreeSet<T> {
         Recover::replace(&mut self.map, value)
     }
 
-    /// Removes a value from the set. Returns `true` if the value was
+    /// Removes a value from the set. Returns whether the value was
```

Comment:
You've changed a bunch of "returns whether" into "returns `true` if", and now the other way round; what's the rationale here?

Reply:
Hmm, there shouldn't be any `Returns whether` left there. I previously normalised in favour of that, but it seems that public opinion was against me (despite the logic I offered), so I've changed back to make folks happy... evidently not completely.